wp-admin/js/widgets/media-image-widget.js

```diff
@@ -106,7 +106,7 @@
 					image_title: '',
 					link_classes: '',
 					link_rel: '',
-					link_url: attachment.linkUrl ? attachment.linkUrl : attachment.link,
+					link_url: attachment.linkUrl,
```

Comment:
@obenland that should work, but it would seem to be somewhat redundant. If the `link_type` is `file` then the PHP logic isn't going to look at `link_url` anyway, and will instead look at `url` anyway, per `\WP_Widget_Image::render_media()`:

```php
$url = '';
if ( 'file' === $instance['link_type'] ) {
	$url = $attachment ? wp_get_attachment_url( $attachment->ID ) : $instance['url'];
} elseif ( $attachment && 'post' === $instance['link_type'] ) {
	$url = get_attachment_link( $attachment->ID );
} elseif ( 'custom' === $instance['link_type'] && ! empty( $instance['link_url'] ) ) {
	$url = $instance['link_url'];
}
```